Maquetado/index.html

```diff
@@ -10,10 +10,10 @@
 </head>
 
 <body>
-  <div class='front-end'>
-    <p class='text'>Front-End Tools</p>
+  <div class="title">
+    <img class="img">
+    <h2 class="text">Front-End Tools</h2>
```

Comment:
I would suggest you not so generic class names as it is very probable that you would make more 'title's on the app. Encapsulate them with specific names, like, 'header', 'header-title', 'header-img', 'header-text'.